doc/logging.md

```diff
@@ -0,0 +1,17 @@
+# How does Happy test log work
+
+## Types of Happy logs
+  * Happy debug logs — Happy will run as test coordination and orchestrator, facilitating the activities of the virtual test device nodes. Happy debug logs will have logs from Happy tool only.
```

Comment:
Let's reword this a little so we're not repeating "Happy" so much

Debug logs - Logs related to Happy's role as test coordinator and orchestrator, where it facilitates the activities of virtual device nodes.

Reply:
OKay

Reply:
done